spec/requests/api/querying_spec.rb

```diff
@@ -643,6 +643,53 @@ def create_vms_by_name(names)
       expect(actions.first["name"]).to eq("suspend")
     end
 
```

Comment:
Can you add a test on actions returned for the collection itself /api/vms,  maybe authorize with vm_start and vm_stop.  We seem to be missing that test.